### PR TITLE
Welcome screen sections renaming

### PR DESCRIPTION
**Pull request by pmlodawski**
### Pull Request Description

- This PR does some improvements to Welcome Screen:
- It removes not implemented yet Community section
- Renames "Tutorial" section to "Sample Projects"
- Renames "Private" section to "My Projects"

### Checklist
Please include the following checklist in your PR:

- [ ] The documentation has been updated if necessary.
- [ ] The code conforms to the [Luna Style Guide](https://github.com/luna/wiki/blob/master/code-style/01.general.md) if it is Haskell.
- [ ] The code has been tested where possible.

### CODE REVIEW

**Comment by mikusp** [comment]
What about "open project" button?

**Comment by pmlodawski** [comment]
I'm adding separate task for this: #1186

File: luna-studio/atom/lib/project-item.js [hunk 1]
```diff
@@ -56,8 +56,7 @@ module.exports = {
                     </div>
         }
     },
-    tutorialClasses: projectClasses + "luna-welcome__tile--tutorial",
+    sampleProjectClasses: projectClasses + "luna-welcome__tile--sample-project",
```
Comment (piotrMocz) [inline luna-studio/atom/lib/project-item.js]: Don't we need to update the CSS files as well?

Comment (piotrMocz) [reply luna-studio/atom/lib/project-item.js]: Okay, I was made aware that this class is not used anywhere (it was a future-proofing kind of thing).

**Pull request merged**